### DISCUSSION TITLE
Asking for PA suggestions

### DISCUSSION THREAD

**Discussion by aibtw** [srsran/srsRAN_Project#366] [General Help] [answered]
Hi all,

I am using the following setup:

Software: 
- Ubuntu 22.04
- srsgnb (srsRAN Project release 23.10.1) 
- open5gs (release 2.6.6)

Hardware: 
- i7 13700k CPU/ 32GB RAM
- B210 USRP
- Multiple COTS UEs (realme 9i 5G, Galaxy A14 5G, Galaxy A53 5G, all of which I found working with srsgnb)

I am trying to achieve some higher range with my b210 usrp (about +100 meters). The poor B210 won't achieve this on its own so I need some power amplification, and I am having trouble deciding on a suitable solution.

Anyone tried something similar? Anyone has suggestions for what PA could work with the B210 in the CBRS band?

**Comment by andrepuschmann** [comment]
We've not used any PA ourselves but I know that the folks in Utah/POWDER use PA in conjunction with the B210. Perhaps @dustinmaas can share infos about the specific model

**Reply by dustinmaas** [reply, marked as answer]
@aibtw I don't have good recommendation for a PA part at this point. We did design/build a TDD RF front end for CBRS use, which relies on a 4W PA from Skyworks (SKY66293-21). We have several of these working in the field, but we've seen too many failures with that part to recommend using it in other designs. (It's not clear which band(s) you want to operate in; maybe CBRS isn't your target anyway. Also note that this is not a connectorized part, in case that is what you are looking for.) We are currently exploring other options for the next revision of our front end, including designing our own transistor-based PA, but we haven't landed on a final design yet.
